pkg/flare/archive_nix.go

```diff
@@ -64,25 +64,30 @@ func (p permissionsInfos) statFiles() error {
 			return fmt.Errorf("can't retrieve file %s uid/gid infos: %s", filePath, err)
 		}
 
-		u, err := user.LookupId(strconv.Itoa(int(sys.Uid)))
+		var uname string
+		var uid = strconv.Itoa(int(sys.Uid))
+		u, err := user.LookupId(uid)
 		if err != nil {
-			return fmt.Errorf("can't lookup for uid info: %v", err)
-		}
-		g, err := user.LookupGroupId(strconv.Itoa(int(sys.Gid)))
-		if err != nil {
-			return fmt.Errorf("can't lookup for gid info: %v", err)
+			uname = uid
```

Comment:
worth mentioning in a comment that this is expected when the user no longer exists on the system